rhizome/host/bin/delete-old-serial-logs

```diff
@@ -0,0 +1,31 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require "fileutils"
+
+directory_path = "/var/log/ubicloud/serials"
+
+# Delete files older than 1 hour
+Dir.glob(File.join(directory_path, "*")).each do |file|
+  next unless File.file?(file) # Skip directories
+
+  file_age_in_hours = (Time.now - File.mtime(file)) / 3600
+  FileUtils.rm(file) if file_age_in_hours > 1
+end
+
+# Reduce the directory size to less than 1GB
+total_directory_size = Dir.glob(File.join(directory_path, "**", "*")).select { |file| File.file?(file) }.sum { |file| File.size(file) }
+files_sorted = Dir.glob(File.join(directory_path, "*"))
+  .select { |file| File.file?(file) }
+  .sort_by { |file| File.size(file) }
+while total_directory_size > 1 * 1024 * 1024 * 1024 # 1GB in bytes
+  largest_file = files_sorted.pop
+  if largest_file.nil?
+    exit
+  end
+
+  file_size = File.size(largest_file)
+  total_directory_size -= file_size
+
+  FileUtils.rm(largest_file)
+end
```

Comment:
nit: 
- We don't need to call `File.size` third times.
- I prefer `break` the loop instead of `exit` the script.
```suggestion
files_with_sizes = Dir.glob(File.join(directory_path, "*"))
  .select { File.file?(_1) }
  .map { {file: _1, size: File.size(_1)} }
  .sort_by { _1[:size] }

while files_with_sizes.sum { _1[:size] } > 1 * 1024 * 1024 * 1024 # 1GB in bytes
  break unless (largest_file = files_with_sizes.pop)
  FileUtils.rm(largest_file[:file])
end
```